velox/docs/functions/spark/string.rst

```diff
@@ -179,6 +179,18 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT substring('Spark SQL', -10, 3); -- "Sp"
         SELECT substring('Spark SQL', -20, 3); -- ""
 
+.. spark:function:: substring_index(string, delim, count) -> varchar
+
+    Returns the substring from ``string`` before ``count`` occurrences of the delimiter ``delim``.
+    If ``count`` is positive, everything to the left of the final delimiter
```

Comment:
Is count = 0 allowed? If so, what is the behavior in that case?

Reply:
Added count = 0 case.